pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/buffer/TransactionBufferCloseTest.java

```diff
@@ -61,7 +61,7 @@ public Object[][] isPartition() {
         };
     }
 
-    @Test(timeOut = 10_000, dataProvider = "isPartition")
+    @Test(dataProvider = "isPartition")
```

Comment:
This seems like it belongs in a separate PR.

Reply:
rollbacked